iceoryx_hoofs/include/iceoryx_hoofs/internal/cxx/storable_function.inl

```diff
@@ -125,16 +142,19 @@ inline storable_function<Capacity, signature<ReturnType, Args...>>::~storable_fu
     m_operations.destroy(*this);
 }
 
+// AXIVION Next Construct AutosarC++19_03-A2.10.1: false positive, args does not hide anything
 template <uint64_t Capacity, typename ReturnType, typename... Args>
 inline ReturnType storable_function<Capacity, signature<ReturnType, Args...>>::operator()(Args... args) const noexcept
 {
+    // AXIVION Next Construct AutosarC++19_03-M0.3.1: m_invoker is initialized in ctor or assignment,
+    // can only be nullptr if this was moved from (calling operator() is illegal in this case)
```

Comment:
Previously a `cxx::Expects(!empty())` was here. I think it's still needed here as a user could invoke a moved-from `cxx::function` and then `m_invoker` would be a nullptr. Can you please re-add it?

Reply:
We can add an an assert here but no always enabled runtime check for those kind of misuse.